pallets/identity-management/src/lib.rs

```diff
@@ -81,6 +81,10 @@ pub mod pallet {
 		// TODO: do we need account as event parameter? This needs to be decided by F/E
 		LinkIdentityRequested {
 			shard: ShardIdentifier,
+			account: T::AccountId,
```

Comment:
Do we need to update something on the clients/sdk for this ? @jonalvarezz

Reply:
Thanks @silva-fj , I checked it up, and can confirm the type isn't use on the client-sdk. Furthermore, client-sdk doesn't not really on parachain other than for fetching metadata and registered enclave data.